README.md

```diff
@@ -20,6 +20,8 @@ Features:
 - Pretty much every part of the table is customizable (borders, lines, padding, alignment).
 - Constraints on columns that allow some additional control over how to arrange content.
 - Cross plattform (Linux, macOS, Windows).
+- It's FAST!
+    Benchmarks show that a pretty big table with weird constraints is build in 4 _us_, which is `4*1e^6`.
```

Comment:
Here's the unicode char for micro seconds
```suggestion
    Benchmarks show that a pretty big table with weird constraints is build in 4 _μs_, which is `4*1e^6`.
```

Reply:
Thanks :)